testing/floydWarshall.py

```diff
@@ -25,12 +28,19 @@ def getShortestPath(
         for i in range(numMuni):
             adjMatrix[i][i] = 0
 
+        #create array of 100 muni codes
+        arrayOfMuniCodes = []
```

Comment:
This can be changed to setOfMuniCodes= set(). Then, when you add to it, do setOfMuniCodes.add(muni.code). This will give it constant time lookups and ensure there is not added overhead with each check to see if it is part of the graph.